File: app/src/androidTest/java/app/we/go/emojidraw/arch/di/TestPracticeComponent.java
package app.we.go.emojidraw.arch.di;

import app.we.go.emojidraw.arch.di.features.practice.PracticeComponent;
import dagger.Subcomponent;

@ActivityScope
@Subcomponent(modules = {TestPracticeModule.class})
public interface TestPracticeComponent extends PracticeComponent{

    @PracticeDuration
    Integer practiceDuration();
}

File: app/src/androidTest/java/app/we/go/emojidraw/arch/di/TestPracticeComponent.kt
package app.we.go.emojidraw.arch.di

import app.we.go.emojidraw.arch.di.features.practice.PracticeComponent
import dagger.Subcomponent

@ActivityScope
@Subcomponent(modules = [TestPracticeModule::class])
interface TestPracticeComponent : PracticeComponent {

    @PracticeDuration
    fun practiceDuration(): Int?
}
